lib/simple-dom/document.js

```diff
@@ -145,22 +145,32 @@ if(Object.defineProperty) {
 	});
 }
 
-function addMissingMethod(name){
-	if (Document.prototype[name])return
+function warnMissingMethod(name, methodOrNot = true){
 	var hasWarned = false;
-	Object.defineProperty(Document.prototype, name, {
-		get: function(){
-			if(!hasWarned) {
+	var warning = "document."+name+" is not implemented in can-simple-dom with witch SSR occurs.  Please make sure this code runs after the document has finished loading, with guide at bitballs (turning off SSR) under donejs.com."
+	if (methodOrNot){
+		hasWarned = true
+		Document.prototype[name] = function(){
+			if(!hasWarned){
 				hasWarned = true;
-				console.warn("document."+name+"() is not implemented in can-simple-dom with witch SSR occurs.  Please make sure this code runs after the document has finished loading, with guide at bitballs (turning off SSR) under donejs.com.")
+				console.warn(warning)
 			}
-			return
-		},
-	})
+		}
+	} else {
+		Object.defineProperty(Document.prototype, name, {
+			get: function(){
+				if(!hasWarned) {
+					hasWarned = true;
+					console.warn(warning)
+				}
+				return
+			},
```

Comment:
I think a setter is always required. I would actually make the getter/setter pair work and save the local value within this function's scope. This way if someone added `querySelector` to the document, it would work thereafter.